android/filament-android/src/main/java/com/google/android/filament/js/Bind.java

```diff
@@ -0,0 +1,6 @@
+package com.google.android.filament.js;
+
+// Java bindings for UEXGL.h interface
+public class Bind {
+    public static native void BindToContext(long jsCtxPtr);
```

Comment:
this is the JNI interface i use to tell bind code to fill the bindings on a specific javascript context